### PR TITLE
Atilla/in 497/fix date without timezone in inbound patientpush

### PR DESCRIPTION
**Pull request by atillacarbon**
## Purpose
✅  If a date contains timezone info in the ccda file redox converts this date correctly.
✅  If a date contains time, redox can calculate right date time with Source's timezone setting. 
😡  However, Redox doesn't add timezone to a `date without time`. In an other words, If a date in ccda is in `yyyyMMdd` format redox just passes this date as it is with `yyyy-MM-dd` format.

In our scala-redox lib we have joda.time.DateTime type for date fields. If date is in `yyyy-MM-dd` format, then our lib convert this date in UTC timezone. This causes 6 hours gap for our Colorado import because of timezone (Denver is GMT-6). 

## Approach
We changed all `DateTime`s to `String`s in our inbound PatientPush and we will calculate right epoch in our api using acquired clinic timezone (we will add timezone field into  ccdaImportJob). I will open another PR for api changes.

## Learning
https://developer.redoxengine.com/onboarding/

image

### CODE REVIEW

File: src/main/scala/com/github/vitalsoftware/scalaredox/models/clinicalsummary/inbound/PatientPush.scala [hunk 1]
```diff
@@ -1,11 +1,8 @@
 package com.github.vitalsoftware.scalaredox.models.clinicalsummary.inbound
 
 import com.github.vitalsoftware.macros.jsonDefaults
-import com.github.vitalsoftware.scalaredox.models.{ Allergy, Insurance, Meta, Problem }
-import com.github.vitalsoftware.scalaredox.models.clinicalsummary.{ ClinicalSummaryLike, Header }
+import com.github.vitalsoftware.scalaredox.models.Meta
 import com.github.vitalsoftware.util.RobustPrimitives
-import play.api.libs.json.JodaWrites._
-import play.api.libs.json.JodaReads._
```
Comment (talha-sen-carbon) [inline src/main/scala/com/github/vitalsoftware/scalaredox/models/clinicalsummary/inbound/PatientPush.scala:8]: Did you remove these when applying context action? These should not be removed I think.

Comment (atillacarbon) [reply src/main/scala/com/github/vitalsoftware/scalaredox/models/clinicalsummary/inbound/PatientPush.scala:8]: I think we don't need `Joda` related things anymore as we won't use Joda DateTime's

Comment (talha-sen-carbon) [reply src/main/scala/com/github/vitalsoftware/scalaredox/models/clinicalsummary/inbound/PatientPush.scala:8]: Oh I see yeah. Ok then this seems fine.

**Review by talha-sen-carbon — DISMISSED** [review]
LGTM 🚀

**Review by talha-sen-carbon — APPROVED** [review]
LGTM

**Pull request merged**